deploy/README.md

```diff
@@ -0,0 +1,214 @@
+
+## Setting up JupyterHub
+Follow this documentation to setup a basic JupyterHub installation. Customizations for the PAOTM course will follow in the next sections. The config.yaml in git already contains some settings for the customizations. For this step the basic config.yaml from the documentations can be used.
+
+[Setting up JupyterHub](https://zero-to-jupyterhub.readthedocs.io/en/latest/setup-jupyterhub/setup-jupyterhub.html)
+
+## Create custom container
+For the course we can make a custom container. The container contains the model and notebook for the course.
+First we need a private container registry to store the container. Create a registry on microsoft Azure with the follow commands:
+Resource group should be the same as the kubernetes cluster.
+
+    az acr create --resource-group <resource group> --name <name container registry> --sku Basic
+
+Login the container registry
+
+	az acr login --name <name container registry>
+
+Attach the container registry to the kubernetes cluster
+
+	az aks update -n <cluster-name> -g <resource group> --attach-acr <name container registry>
+
+The dockerfile for the custom container is stored in thi git repo: Dockerfile-jupyterhub. You first need to download FM from the [build server.](https://build.deltares.nl/buildConfiguration/Dimr_DimrCollector) [This build](https://build.deltares.nl/repository/download/Dimr_DimrCollector/1306704:id/dimrset_lnx64_373.tar.gz) was used for the demo.
+
+ - Download FM, unzip the file and store it in the models folder. 
+ - Change the reference in Dockerfile-jupyter to the model in the models folder. It is something like [container registry name].azurecr.io
+
+Build the container localy. The loginServer is a result from the creation of the container registry
+
+	docker build -f Dockerfile-jupyterhub -t <loginServer>/hydrodynamicscourse:<version number> .
+
+Push the container to the azure container registry
+
+	docker push <loginServer>/hydrodynamicscourse:v1
+
+Update the JupyterHub config.yaml with the new container. Add the following code block:
+
+	singleuser:
+	  image:
+	    name: loginServer<>/hydrodynamicscourse
+	    tag: <version>
+
+Update the helm installation:
+
+	 RELEASE=jhub
+	 helm upgrade $RELEASE jupyterhub/jupyterhub --version=0.8.2 -f config.yaml
+
+## Customize resources
+The resources for a container can be specified in the config.yaml
+
+	singleuser:
+	  image:
+	    name: loginServer<>/hydrodynamicscourse
+	    tag: v4
+	  memory:
+	    limit: 4G
+	    guarantee: 4G
+	  cpu:
+	    limit: 1.5
+	    guarantee: 1.5
+
+Again update the helm installation:
+	
+	RELEASE=jhub
+	 helm upgrade $RELEASE jupyterhub/jupyterhub --version=0.8.2 -f config.yaml
+
+## Customize storage
+
+By default JupyterHub creates a persistent volume (pv) en persistent volume claim (pvc) for each user. The volume is mounted in /home/jovyan. So all the changes here will be stored in the persistent volume.
```

Comment:
jovyan? 🤭

Reply:
That's the default username in jupyter (jovyan == inhabitant of the planet Jupyter)